openxlsx.Rmd

```diff
@@ -0,0 +1,581 @@
+
+</li>
+
+<li>
+
+Some sections have multiple parts to them. Navigate the course by using the buttons at the bottom of the screen to Continue or go to the Next Topic.
+
+</li>
+
+<li>
+
+The course will also show progress through sections, a green tick will appear on sections you've completed, and it will remember your place if you decide to close your browser and come back later.
+
+</li>
+
+</ul>
+
+There are some examples designed to accompany this course, these can either be completed bit by bit as you work through the course or as a whole at the end.  There is also an exercise for you to try when you finish the course to apply what you've learned. Both can be downloaded from [Github](https://github.com/Public-Health-Scotland/).
+  </div>
+</br>
+
+
+#### What is openxlsx?
+
+Openxslx is an R package that allows us to read, write and edit xlsx files from within the RStudio interface. This will let us automate the creation and formatting of the excel files that usually accompany our publications. 
+
+The process to create these excel publication files is usually very manual and involves several copy/paste steps. This can result in errors with data being pasted in the wrong place or wrong format which can have serious consequences. For example, in 2003 the Candian power company TransAlta lost $24 million due to a copy paste error causing misaligned rows in a spreadsheet containing bids for contracts. For more examples of excel errors see this [blog post on LinkedIn](https://www.linkedin.com/pulse/7-biggest-excel-mistakes-all-time-andrew-hoag).
+
+
+### Why use Excel?
+You might wonder why we should still provide data in an excel format, aren't dashboards and infographics the way forward? 
+
+While dashboards and infographics are a great way of showing results and delivering messages they don't suit everyone. Many of our customers can't access dashboards due to IT setup in their organisations and others may just want the data in a spreadsheet to take away and do their own analysis. 
+
+```{r openxlsx-horst, echo=FALSE, fig.align='centre', out.width="100%"}
+knitr::include_graphics("images/openxlsx/openxlsx_horst.jfif")
+```
+
+
+## Creating a workbook
+We will start by creating a blank workbook and then slowly build it up by adding tables, charts styles and formatting. 
+
+Let's start by creating a blank excel template that you build on. Note that you can't view it as you go along, only once it is saved out, so you have to keep track of what you are adding. 
+
+````
+wb <- createWorkbook()
+````
+
+### Adding worksheets
+Before we can add any data to the workbook we have to create worksheets for the data to go in. Note that the order you add the sheets to the workbook is the order they appear in, this is not set in code. We will add 3 worksheets:
+
+* A notes page
+* A data page
+* A charts page
+
+Notes page:
```

Comment:
You could use low-level headings which will help structure the page and make it easier to spot the different sections. Perhaps:

```suggestion
#### 1 - Notes page
```

[...]
`#### 2 - Data page`
[...]
`#### 3 - Charts page`